.github/workflows/web-app-docker-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Models web app Docker Publisher
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+
+env:
+  IMAGE_NAME: models-web-app
```

Comment:
@animeshsingh Any preference with the image name?

Reply:
`models-web-app` works for me.

Reply:
@animeshsingh I'll respond here to your comment https://github.com/kubeflow/kfserving/pull/1511#issuecomment-815306317 to keep the conversation in one place.

I don't have a strong opinion on the name. The only reason I named it `models-web-app` was because I was thinking of placing the app under `Models` in the KF CentralDashboard in the future. 

I think the argument for shorter names is a good one. Looking at the repo folders and the docker images I see that  all of the components are named as one word. So it would make sense to keep this pattern and name it just `ui`, or something similar. The docker image would end up being `kfserving/ui`. We also follow this code naming with Katib [[code](https://github.com/kubeflow/katib/tree/master/pkg/ui)].

@yuzisun @pvaneck @animeshsingh what do you think? If you'd prefer the `ui` name I can make the change to the code as well. If you don't have a hard preference I guess we can keep `models-web-app`(?)

Reply:
@yuzisun @pvaneck @animeshsingh `models-web-app` it is then?